### PR TITLE
Small fixes for geiger_counter

### PR DESCRIPTION
**Pull request by mchindri**

### CODE REVIEW

File: projects/ADuCM3029_demo_cn0536/app_src/communication.h [hunk 1]
```diff
@@ -79,8 +79,13 @@
 #define MQTT_CLIENT_NAME	"analog_client"
 #define MQTT_PUBLISH_TOPIC	"analog_test_topic"
 
-int32_t init_communication(struct uart_desc **desc,
+typedef union {
```
Comment (adrimbarean) [inline projects/ADuCM3029_demo_cn0536/app_src/communication.h:82]: I'd still not use typedef here.

**Comment by mchindri** [comment]
v2:
 - Fix astyle

File: projects/ADuCM3029_demo_cn0536/app_src/main.c [hunk 2]
```diff
@@ -56,6 +57,58 @@
...
+
+typedef enum {
+	STATUS_SETUP,
+	STATUS_OK,
+	STATUS_ERR
+} status_t;
+
+struct gpio_desc *green_led;
+struct gpio_desc *blue_led;
+
```
Comment (adrimbarean) [inline projects/ADuCM3029_demo_cn0536/app_src/main.c:73]: nitpick: extra line

File: projects/ADuCM3029_demo_cn0536/app_src/main.c [hunk 3]
```diff
@@ -56,6 +57,58 @@
 /* Size of the buffer used to send data */
 #define DATA_BUFF_SIZE		256
 
+#define GREEN_LED_NB		32
+#define BLUE_LED_NB		31
+#define LED_ON			GPIO_HIGH
+#define LED_OFF			GPIO_LOW
+
+typedef enum {
```
Comment (adrimbarean) [inline projects/ADuCM3029_demo_cn0536/app_src/main.c:65]: If not necessary, I'd avoid typedef.

**Review by adrimbarean — APPROVED** [review]
Looks good to me. I had a few comments, but can live with them.

**Comment by mchindri** [comment]
v3:
 - Remove extra line
 -  Remove typdefs

**Pull request merged**